### PR TITLE
feat(suite-native): enable Solana via feature flag

### LINKED ISSUE #14895: Solana - create feature flag
BY enabling feature flag, user should be able to:

- enable Solana and discover accounts
- import Solana to Portfolio Tracker
- See SPL tokens and txns for the account

### PR DESCRIPTION
**Pull request by vytick**
Add Solana to mobile via feature flag and make it discoverable

## Related Issue

Resolve #14895

### CODE REVIEW

File: suite-native/app/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'react-native-get-random-values';
```
Comment (PeKne) [inline suite-native/app/index.js]: Could not we use AI for this? 😅

Just joking. But why is this needed? You are not importing anything specific from this package. Is it some polyfill that Solana uses in the background? If so, add some comment explaining that please

Comment (vytick) [reply suite-native/app/index.js]: It has react native implementation of `crypto.getRandomValues()`. Otherwise Solana api wont work

Comment (PeKne) [reply suite-native/app/index.js]: Maybe it should be placed in the metro.config.ts, then. We are already polyfilling the `crypto` module there. Cc @Nodonisko, what is your opinion about it?

Comment (Nodonisko) [reply suite-native/app/index.js]: I don't like it, we should either use polyfill that we already have (it's only JS so it will be slow) or use `react-native-quick-crypto` which is estabilished lib a will want to use it in future anyway.

Comment (Nodonisko) [reply suite-native/app/index.js]: If we need only that one function we already using expo-crypto in app which has getRandomValues too https://docs.expo.dev/versions/latest/sdk/crypto/#cryptogetrandomvaluesttypedarray

Comment (vytick) [reply suite-native/app/index.js:1]: I just need get polyfill for crypto.getRandomValues and this one does just this one thing, has bunch of installs (900k) and is suggested by solana mobile for expo: https://docs.solanamobile.com/react-native/expo

Comment (vytick) [reply suite-native/app/index.js]: updated in c92279d039

Comment (vytick) [reply suite-native/app/index.js]: Fyi: I have a followup branch to get SOL data for `getAccountHistoryMovementFromTransactions`, but I dont want to make this PR too complex

**Review by PeKne — APPROVED** [review]
